src/lib/chips/chip-list.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  ElementRef,
+  ModuleWithProviders,
+  NgModule,
+  QueryList,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {MdChip} from './chip';
+
+@Component(MdChipList.COMPONENT_CONFIG)
+export class MdChipList {
+
+  static COMPONENT_CONFIG: Component = {
+    moduleId: module.id,
+    selector: 'md-chip-list, [md-chip-list]',
+    template: `<ng-content></ng-content>`,
+    host: {
+      // Properties
+      'tabindex': '0',
+      'role': 'listbox',
+    },
+    queries: {
+      items: new ContentChildren(MdChip)
+    },
+    styleUrls: ['chips.css'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+  };
+
+  protected items: QueryList<MdChip>;
+
+  constructor(private _elementRef: ElementRef) {}
+
+  ngAfterContentInit(): void {
+    this._elementRef.nativeElement.classList.add('md-chips');
```

Comment:
You should be able to add `class` to the `host` oject in the component metadata for this:
```ts
host: {
  'class': 'md-chip-list'
}
```
Same for chip below.